LiskHQ/lisk-contracts
# Add script to deploy OptimismMintableERC20 USDT token using OptimismMintableERC20Factory

### What was the problem?

This PR resolves #153

### How was it solved?

Added script with environment variable for testnet and mainnet

### How was it tested?

 - Run the script from inside scripts folder `bash 6_deployUSDTContract.sh `
 - Add `--nonce` flag to specify the correct nonce in the command in script, in case there is a nonce mismatch reported
 - Supply private key at the prompt to sign and send the transaction
 - Wait for the process to finish, check the output transaction hex in block explorer
